### ISSUE TITLE
Edit auto reply email

### ISSUE THREAD

**Issue by Muraszko** [afred/openvault#1185]
The auto reply email sent when a user requests needs to have more information within it.
I'm not sure if users currently are receiving an email.  I just tested the system with two email addresses and I haven't received any emails.

Once that's working, it should be sending out the following email response.
```
Hello,
Thank you for your interest in the digitization of WGBH Open Vault artifact <TITLE AND URL OF ARTIFACT HERE>

With Open Vault, we've added the ability to
request digitization of transcripts, audio or video.  If you agree to the pay
the digitization fee and we're able to (rights pending), WGBH will make
available the digitized audio, video and/or transcript for public access on the
Open Vault website.

If you would like to pay for the cost of digitizing, or would like more information, please email openvault@wgbh.org and we will then provide you
with a more accurate estimation of what the cost will be.

If you wish to withdraw your request for digitization of this asset,
you may do so via your profile page on Open Vault.  If we receive no
response from you for more information regarding your requests within 30 days, the requests will be
withdrawn from your profile.

If you wish for information about licensing footage from a WGBH program please contact
WGBH Stock Footage Sales at stock_sales@wgbh.org or 617-300-3939.

If you wish to view WGBH material that is not available on Open Vault, you may schedule an appointment to visit the WGBH Media Library and Archives department in Boston, Massachusetts by emailing archive_requests@wgbh.org.

Thank you for your interest in WGBH Open Vault.  Please let us know if you
have any questions.

WGBH Media Library and Archives
openvault@wgbh.org
One Guest St.
Boston, MA 02135
```

**DONE WHEN**
The auto reply email contains the above text and is emailing users who have made the request for digitization.

**Comment by afred** [comment]
@Muraszko what should the subject be? Currently setting it to: "WGBH Digitization Request Received", but can easily be changed.

**Comment by Muraszko** [comment]
@afred 
You can leave it that.  Thanks!

**Comment by afred** [comment]
fixed by #1201 #1203. @Muraszko try it out on staging, and close this ticket if it looks ok. Note that Staging and Production currently share databases, so requests made on Staging will show up on Production.

**Comment by Muraszko** [comment]
@afred :+1: on staging.
